Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Lights/Ibl.azsli

```diff
@@ -85,12 +85,12 @@ void ApplyIBL(Surface surface, inout LightingData lightingData)
 
     if(useIbl)
     {
-        float iblExposureFactor = pow(2.0, SceneSrg::m_iblExposure);
+        float exposure = pow(2.0, ObjectSrg::m_reflectionProbeData.m_exposure);
```

Comment:
Is SceneSrg::m_iblExposure being removed entirely? Or only in some places?

Reply:
It's still used for global IBL, this is only for ReflectionProbes.